lib/task-data/tasks/catalog-perccli.js

```diff
@@ -0,0 +1,23 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'Catalog perccli',
+    injectableName: 'Task.Catalog.perccli',
+    implementsTask: 'Task.Base.Linux.Catalog',
+    options: {
+        adapter: '0',
```

Comment:
Show all commands should work. However most servers and storage only have one controller. Not sure if Ben have any comments here. @benbp